.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
Looks good ! I'm not sure if we need all of it but I found a gitignore for react [here](https://github.com/facebook/react/blob/main/.gitignore) that had a bit more stuff. Especially the .vs code. Maybe worth adding that in?